.snippets/code/develop/parachain-devs/runtime-development/FRAME/add-pallet-to-the-runtime/frame-support-runtime-macro.rs

```diff
@@ -0,0 +1,21 @@
+#[frame_support::runtime]
+mod runtime {
+	#[runtime::runtime]
```

Comment:
Is this amount of spacing normal? It seems like in other places it's only 4 spaces, but I don't know about Rust so, just curious 🙂

Reply:
Well I just checked on VSCode and it's 4 spaces as well, also I run cargo `rustfmt` just in case

Reply:
There's an issue with whitespace in this file. There are tabs instead of 4 spaces.